Rebase PR#10798 Update standard EventContent definitions

jeongeun opened the pull request:
"Fix Conflicts"

jeongeun committed:
Update RecoMET_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to add missing Module ('keep recoPFMETs_pfMetEI_*_*',)
to fix Label name ('keep recoMuonMETCorrectionDataedmValueMap_muonMETValueMapProducer_muCorrData_*',)
and change the wildcards to specific Type and Modules (#'keep *HaloData_*_*_*',)

jeongeun committed:
Update RecoMET_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to add missing Module ('keep recoPFMETs_pfMetEI_*_*',)
to fix Label name ('keep recoMuonMETCorrectionDataedmValueMap_muonMETValueMapProducer_muCorrData_*',)
and change the wildcards to specific Type and Modules (#'keep *HaloData_*_*_*',)

jeongeun committed:
Update RecoMET_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to add missing Module ('keep recoPFMETs_pfMetEI_*_*',)
to fix Label name ('keep recoMuonMETCorrectionDataedmValueMap_muonMETValueMapProducer_muCorrData_*',)
and change the wildcards to specific Type and Modules (#'keep *HaloData_*_*_*',)

jeongeun committed:
Update RecoMET_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to add missing Module ('keep recoPFMETs_pfMetEI_*_*',)
to fix Label name ('keep recoMuonMETCorrectionDataedmValueMap_muonMETValueMapProducer_muCorrData_*',)
and change the wildcards to specific Type and Modules (#'keep *HaloData_*_*_*',)

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMuon_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove lines (no such Modules in EventContent)

jeongeun committed:
Update RecoMuon_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove lines (no such Modules in EventContent)

jeongeun committed:
Update RecoMuon_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove lines (no such Modules in EventContent)

jeongeun committed:
Update RecoMuon_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove lines (no such Modules in EventContent)

jeongeun committed:
Update muIsolation_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove some line ( no such Module in EventContent)

jeongeun committed:
Update muIsolation_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove some line ( no such Module in EventContent)

jeongeun committed:
Update muIsolation_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove some line ( no such Module in EventContent)

jeongeun committed:
Update muIsolation_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoTauTag_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove wildcards and replaced with specific module names
#'keep *_hpsPFTauDiscrimination*_*_*', --> 'keep *_hpsPFTauDiscriminationByDeadECALElectronRejection_*_*' .. etc
#'keep *_hpsPFTau*PtSum_*_*',  -->  'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*', .. etc

jeongeun committed:
Update RecoTauTag_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove wildcards and replaced with specific module names
#'keep *_hpsPFTauDiscrimination*_*_*', --> 'keep *_hpsPFTauDiscriminationByDeadECALElectronRejection_*_*' .. etc
#'keep *_hpsPFTau*PtSum_*_*',  -->  'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*', .. etc

jeongeun committed:
Update RecoTauTag_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to remove wildcards and replaced with specific module names
#'keep *_hpsPFTauDiscrimination*_*_*', --> 'keep *_hpsPFTauDiscriminationByDeadECALElectronRejection_*_*' .. etc
#'keep *_hpsPFTau*PtSum_*_*',  -->  'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*', .. etc

jeongeun committed:
Update RecoVertex_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
 and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoVertex_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
 and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoVertex_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
 and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix module name
and to remove some lines.(no such Module in EventContent)

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix module name
and to remove some lines.(no such Module in EventContent)

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix module name
and to remove some lines.(no such Module in EventContent)

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix module name
and to remove some lines.(no such Module in EventContent)

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module names
and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module names
and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module names
and remove some line ( no such Module in EventContent)

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module names
and remove some line ( no such Module in EventContent)

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
add Type name and remove some lines ( no such Modules in event content)

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
add Type name and remove some lines ( no such Modules in event content)

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
add Type name and remove some lines ( no such Modules in event content)

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
add Type name and remove some lines ( no such Modules in event content)

jeongeun committed:
Update EITopPAG_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module name from  pfTausDiscrimination* to pfTausDiscriminationByDecayModeFinding, pfTausDiscriminationByIsolation
and add Type names

jeongeun committed:
Update EITopPAG_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module name from  pfTausDiscrimination* to pfTausDiscriminationByDecayModeFinding, pfTausDiscriminationByIsolation
and add Type names

jeongeun committed:
Update EITopPAG_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions
to change Module name from  pfTausDiscrimination* to pfTausDiscriminationByDecayModeFinding, pfTausDiscriminationByIsolation
and add Type names

jeongeun committed:
Update HLTrigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to add Module or Type names

jeongeun committed:
Update HLTrigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to add Module or Type names

jeongeun committed:
Update HLTrigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to add Module or Type names

jeongeun committed:
Update HLTrigger_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to add Module or Type names

jeongeun committed:
Update Scalers_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to replace with specific Module name and remove wildcard .
#'keep L1AcceptBunchCrossings_*_*_*', --> 'keep L1AcceptBunchCrossings_scalersRawToDigi_*_*',

jeongeun committed:
Update Scalers_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to replace with specific Module name and remove wildcard .
#'keep L1AcceptBunchCrossings_*_*_*', --> 'keep L1AcceptBunchCrossings_scalersRawToDigi_*_*',

jeongeun committed:
Update Scalers_EventContent_cff.py

Update the standard RECO and AOD EventContent definitions 
to replace with specific Module name and remove wildcard .
#'keep L1AcceptBunchCrossings_*_*_*', --> 'keep L1AcceptBunchCrossings_scalersRawToDigi_*_*',

jeongeun committed:
Update MicroEventContent_cff.py

 update the standard event content definitions to have no wildcards of module
#'keep double_fixedGridRho*__*', --> 'keep double_fixedGridRhoAll__*', ...etc
to add specific Module and Label names
Add 'keep *_slimmedMETsPuppi_*_*', 
#'keep recoPhotonCores_reducedEgamma_*_*',  --> 'keep recoPhotonCores_reducedEgamma_reducedGedPhotonCores_*',

jeongeun committed:
Update MicroEventContent_cff.py

 update the standard event content definitions to have no wildcards of module
#'keep double_fixedGridRho*__*', --> 'keep double_fixedGridRhoAll__*', ...etc
to add specific Module and Label names
Add 'keep *_slimmedMETsPuppi_*_*', 
#'keep recoPhotonCores_reducedEgamma_*_*',  --> 'keep recoPhotonCores_reducedEgamma_reducedGedPhotonCores_*',

jeongeun committed:
Update MicroEventContent_cff.py

 update the standard event content definitions to have no wildcards of module
#'keep double_fixedGridRho*__*', --> 'keep double_fixedGridRhoAll__*', ...etc
to add specific Module and Label names
Add 'keep *_slimmedMETsPuppi_*_*', 
#'keep recoPhotonCores_reducedEgamma_*_*',  --> 'keep recoPhotonCores_reducedEgamma_reducedGedPhotonCores_*',

jeongeun committed:
Update RecoJets_EventContent_cff.py

jeongeun committed:
Update RecoJets_EventContent_cff.py

jeongeun committed:
Update RecoJets_EventContent_cff.py

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

jeongeun committed:
Update RecoLocalCalo_EventContent_cff.py

Replacing the wildcard in the Module filed with specific Module

jeongeun committed:
Update RecoLocalCalo_EventContent_cff.py

Replacing the wildcard in the Module filed with specific Module

jeongeun committed:
Update RecoLocalCalo_EventContent_cff.py

Replacing the wildcard in the Module filed with specific Module

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMET_EventContent_cff.py

jeongeun committed:
Update RecoMuon_EventContent_cff.py

jeongeun committed:
Update RecoMuon_EventContent_cff.py

jeongeun committed:
Update RecoMuon_EventContent_cff.py

jeongeun committed:
Update RecoMuon_EventContent_cff.py

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

jeongeun committed:
Update RecoParticleFlow_EventContent_cff.py

jeongeun committed:
Update RecoTracker_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix the Label name.
#'keep *_generalTracks_MVAValues_*', -->  'keep *_generalTracks_MVAVals_*',

jeongeun committed:
Update RecoTracker_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix the Label name.
#'keep *_generalTracks_MVAValues_*', -->  'keep *_generalTracks_MVAVals_*',

jeongeun committed:
Update RecoTracker_EventContent_cff.py

Updating the standard RECO and AOD EventContent definitions
to fix the Label name.
#'keep *_generalTracks_MVAValues_*', -->  'keep *_generalTracks_MVAVals_*',

jeongeun committed:
Update RecoVertex_EventContent_cff.py

jeongeun committed:
Update RecoVertex_EventContent_cff.py

jeongeun committed:
Update RecoVertex_EventContent_cff.py

jeongeun committed:
Update L1Trigger_EventContent_cff.py

jeongeun committed:
Update L1Trigger_EventContent_cff.py

jeongeun committed:
Update L1Trigger_EventContent_cff.py

jeongeun committed:
Update L1Trigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update MicroEventContent_cff.py

jeongeun committed:
Update MicroEventContent_cff.py

jeongeun committed:
Update MicroEventContent_cff.py

jeongeun committed:
Update RecoLocalTracker_EventContent_cff.py

jeongeun committed:
Update RecoLocalTracker_EventContent_cff.py

jeongeun committed:
Update RecoLocalTracker_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update HLTrigger_EventContent_cff.py

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update L1Trigger_EventContent_cff.py

Undo

jeongeun committed:
Update RecoHiJets_EventContent_cff.py

jeongeun committed:
Update RecoHiJets_EventContent_cff.py

jeongeun committed:
Update RecoHiJets_EventContent_cff.py

jeongeun committed:
Update RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Update RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Update RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Update RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Update l1Muon_EventContent_cff.py

jeongeun committed:
Update l1Muon_EventContent_cff.py

jeongeun committed:
Update l1Muon_EventContent_cff.py

jeongeun committed:
Update RecoLocalCalo_EventContentCosmics_cff.py

jeongeun committed:
Update RecoLocalCalo_EventContentCosmics_cff.py

jeongeun committed:
Update RecoLocalCalo_EventContentCosmics_cff.py

jeongeun committed:
Update Scalers_EventContent_cff.py

jeongeun committed:
Update Scalers_EventContent_cff.py

jeongeun committed:
Update Scalers_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun committed:
Merged patch-10 from repository jeongeun

Conflicts:
	RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Merge branch 'merge-attempt' into patch-10

Conflicts:
	RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

jeongeun committed:
Fix confilcts

cmsbuild commented:
A new Pull Request was created by @jeongeun  for CMSSW_7_6_X.

Rebase PR#10798 Update standard EventContent definitions

It involves the following packages:

CalibMuon/RPCCalibration
CommonTools/ParticleFlow
EventFilter/ScalersRawToDigi
PhysicsTools/PatAlgos
RecoEgamma/Configuration
RecoHI/HiEgammaAlgos
RecoHI/HiJetAlgos
RecoJets/Configuration
RecoLocalCalo/Configuration
RecoLocalTracker/Configuration
RecoMET/Configuration
RecoMuon/Configuration
RecoMuon/MuonIsolationProducers
RecoParticleFlow/Configuration
RecoTauTag/Configuration
RecoTracker/Configuration
RecoVertex/Configuration


@diguida, @cvuosalo, @cerminar, @monttj, @cmsbuild, @slava77, @vadler, @mmusich can you please review it and eventually sign? Thanks.
@ghellwig, @TaiSakuma, @yslai, @forthommel, @yduhm, @rappoccio, @Martin-Grunewald, @lgray, @abbiendi, @mmarionncern, @battibass, @makortel, @threus, @acaudron, @jhgoh, @dgulhan, @jdolen, @jlagram, @ferencek, @cerati, @trocino, @rociovilar, @Sam-Harper, @GiacomoSguazzoni, @rovere, @VinInn, @bellan, @nhanvtran, @schoef, @mschrode, @richard-cms, @nickmccoll, @mandrenguyen, @imarches, @gbenelli, @jazzitup, @ahinzmann, @yenjie, @kurtejung, @gpetruc, @istaslis, @mariadalfonso, @yetkinyilmaz, @pvmulder, @bachtis this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@degano you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

jeongeun commented:
merge

cvuosalo commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: 020c2adeb18697e8bcf0b1e080b2064cf4e79c78
 When I ran the RelVals I found an error in the following worklfows: 
 4.22 step2
<pre>runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/step2_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC.log

----- Begin Fatal Exception 02-Sep-2015 19:12:54 CEST-----------------------

An exception of category 'Configuration' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PoolOutputModule label='RECOoutput'

Exception Message:

Invalid statement in configuration file

In OutputModule parameter named 'outputCommands'

In each rule, after 'keep ' or 'drop ' there must

be a branch specification of the form 'type_label_instance_process'

There must be 4 fields separated by underscores

The fields can only contain alphanumeric characters and the wildcards * or ?

Alternately, a single * is also allowed for the branch specification

This is the invalid output configuration rule:

    keep HcalUnpackerReport_castorDigis_*_*keep HcalUnpackerReport_hcalDigis_*_*

Exception thrown from ProductSelectorRules::Rule

----- End Fatal Exception -------------------------------------------------

</pre>
8.0 step2
<pre>runTheMatrix-results/8.0_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS/step2_BeamHalo+BeamHalo+DIGICOS+RECOCOS+ALCABH+HARVESTCOS.log

----- Begin Fatal Exception 02-Sep-2015 19:13:13 CEST-----------------------

An exception of category 'Configuration' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PoolOutputModule label='FEVTDEBUGoutput'

Exception Message:

Invalid statement in configuration file

In OutputModule parameter named 'outputCommands'

In each rule, after 'keep ' or 'drop ' there must

be a branch specification of the form 'type_label_instance_process'

There must be 4 fields separated by underscores

The fields can only contain alphanumeric characters and the wildcards * or ?

Alternately, a single * is also allowed for the branch specification

This is the invalid output configuration rule:

    keep HcalUnpackerReport_castorDigis_*_*keep HcalUnpackerReport_hcalDigis_*_*

Exception thrown from ProductSelectorRules::Rule

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/7836/summary.html

slava77 commented on RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContentCosmics_cff.py:
```diff
@@ -27,9 +27,13 @@
                                            'keep ZDCRecHitsSorted_*_*_*',
-                                           'keep HcalUnpackerReport_*_*_*'
+                                           #'keep HcalUnpackerReport_*_*_*'
+                                           'keep HcalUnpackerReport_castorDigis_*_*'
```
there is a comma missing here.
@jeongeun please double-check all other files and update only after ```runTheMatrix.py -l limited -i all``` completes

cvuosalo commented:
@jeongeun:
Please fix the problems with this pull request.

jeongeun committed:
Update RecoLocalCalo_EventContentCosmics_cff.py

add a comma.

jeongeun committed:
Update muIsolation_EventContent_cff.py

jeongeun commented:
Fix the missing comma and
'runTheMatrix.py -l limited -i all' is completed.
Please test

cmsbuild commented:
Pull request #11068 was updated. @diguida, @cvuosalo, @cerminar, @monttj, @cmsbuild, @slava77, @vadler, @mmusich can you please check and sign again.

slava77 commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 3c09bb00a207b87f2e15228d9d92b9bd39193e9f
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/7948/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/7948/summary.html

slava77 commented:
``` git rebase -i ``` can be used to reduce the number of commits by squashing the intermediate ones

slava77 commented:
the squashing would be nice, but is not absolutely necessary; just trying to keep the history shorter

cvuosalo commented:
@jeongeun:
Your changes have altered the event content, which is not correct. The following collections are missing:

floats_generalTracks_MVAValues
recoPFTauDiscriminator_hpsPFTauDiscriminationByRawPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauDiscriminationByPhotonPtSumOutsideSignalCone
recoPFTauDiscriminator_hpsPFTauDiscriminationByLoosePileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauNeutralIsoPtSum
recoPFTauDiscriminator_hpsPFTauDiscriminationByTightPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauDiscriminationByMediumPileupWeightedIsolation3Hits
recoPFTauDiscriminator_hpsPFTauPUcorrPtSum
recoPFTauDiscriminator_hpsPFTauChargedIsoPtSum

And recoPhotons_gedPhotonsTmp has been incorrectly added.

Please fix these problems.

jeongeun committed:
Update RecoTauTag_EventContent_cff.py

Add missing parts

jeongeun committed:
Update RecoEgamma_EventContent_cff.py

Delete incorrectly added gedPhotonsTmp line

jeongeun commented:
@cvuosalo 
I included the missing parts of hpsPFTau* and deleted the gedPhotonTmp lines.
However in generalTracks_MVAValues case,

When I checked the event content definitions at 7_6_0_pre2
I found the float type generalTracks Module has different Label name "MVAVals" as follow
Type                                                    Module                        Label             Process
---------------------------------------------------------------------------------------------------------------
edm::ValueMap<float>                  "generalTracks"             "MVAVals"         "RECO"

so I changed the Label name "MVAValues" to "MVAVals" in my PR.
Is that label newly updated or changed again? or Do I misunderstand something here?
Please let me know, Thank you.

cmsbuild commented:
Pull request #11068 was updated. @diguida, @cvuosalo, @cerminar, @monttj, @cmsbuild, @slava77, @vadler, @mmusich can you please check and sign again.

cvuosalo commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: ca1d7942ae238471331c25feccfe9d308b0962b4
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/8030/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/8030/summary.html

cvuosalo commented:
@jeongeun:
In the latest version of 76X, CMSSW_7_6_X_2015-09-12-1100, generalTracks_MVAValues is present. This pull request should not remove generalTracks_MVAValues from the event content.

jeongeun committed:
Update RecoTracker_EventContent_cff.py

jeongeun commented:
Yes. I fixed it (not remove generalTracks_MVAValues)
and I'll keep check the event contents in the latest version of 76X. Thank you.
please test

cmsbuild commented:
Pull request #11068 was updated. @diguida, @cvuosalo, @cerminar, @monttj, @cmsbuild, @slava77, @vadler, @mmusich can you please check and sign again.

cvuosalo commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 17a054f45f53fb34e7c03e714625b7dce7c23c8c
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/8062/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11068/8062/summary.html

cvuosalo commented:
+1

For #11068 17a054f45f53fb34e7c03e714625b7dce7c23c8c

Expanding wildcards to be more specific for event content statements. There should be no change in run-time behavior or resulting event content.

The code changes are satisfactory, and Jenkins [tests]() against baseline CMSSW_7_6_X_2015-09-13-1100 show no significant differences, as expected. An extended test of workflow 25202.0_TTbar_13 with 70 events against baseline CMSSW_7_6_0_pre4 shows no significant differences and no change in event content.

slava77 commented:
@monttj @diguida @mmusich 
please check this PR, your signature is needed
Thank you.

slava77 commented:
@monttj @diguida @mmusich
please check this PR, your signature is needed
Thank you.

monttj commented:
+1
It is to make the output contents more specific in AT side.

diguida commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_7_6_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @degano, @smuzaffar

davidlange6 commented:
+1
